OData/test/E2ETest/WebStack.QA.Test.OData/QueryComposition/SelectExpandTests.cs

```diff
@@ -765,7 +883,7 @@ public class SelectOrder
         public int Id { get; set; }
         public DateTimeOffset Date { get; set; }
         public SelectAddress BillingAddress { get; set; }
-        public virtual IList<SelectOrderDetail> OrderDetails { get; set; }
+        public virtual IQueryable<SelectOrderDetail> OrderDetails { get; set; }
```

Comment:
SelectOrder is IEnumerable, this change is for Test both IEnumerable and IQueryable